fashion-messaging-app-api/server.js

```diff
@@ -16,6 +23,27 @@ app.use(cors({
 app.use(express.json()); 
 app.use(morgan('combined')); 
 
+const storage = multer.diskStorage({
+  destination: function(req,file,cb) {
+    cb(null, 'images/')
+  },
+  filename: function(req, file, cb) {
+    cb(null, Date.now() + path.extname(file.originalname));
+  },
+});
+
+const upload = multer({
+  storage: storage,
+  limits: { fileSize: '1000000'},
```

Comment:
why you  limit to  1000000?  any particular reasons ?

Reply:
> why you limit to 1000000? any particular reasons ?

I did because later on, I will have a lot more files posted to my server so I don't want large data files to take up all the space, also to allow my browser load faster and to avoid problems when sending it to my database. I may increase the size to 3-5mb in the future depending on my needs